src/GraphQL.Tests/Validation/ArgumentsOfCorrectTypeTests.cs

```diff
@@ -788,7 +788,7 @@ public void partial_object_missing_required()
                 Rule.badValue(_, "complexArg", "ComplexInput", "{intField: 4}", 3, 33,
                     new[]
                     {
-                        "In field \"requiredField\": Expected \"Boolean!\", found null."
+                        "Missing field 'requiredField'."
```

Comment:
But let's change error message to something like `Missing required field 'requiredField' of type 'Boolean!'.`

It will be event more accurate.

Reply:
```suggestion
                        "Missing required field 'requiredField' of type 'Boolean!'."
```